contrib/ruby_event_store-outbox/lib/ruby_event_store/outbox/runner.rb

```diff
@@ -0,0 +1,40 @@
+module RubyEventStore
+  module Outbox
+    class Runner
+      def initialize(consumer, configuration, logger:)
+        @consumer = consumer
+        @logger = logger
+        @sleep_on_empty = configuration.sleep_on_empty
+        @gracefully_shutting_down = false
+        prepare_traps
+      end
+
+      def init
+        consumer.init
+      end
+
+      def run
+        while !@gracefully_shutting_down
+          was_something_changed = consumer.one_loop
```

Comment:
Consider renaming `consumer.one_loop` to `consumer.call` or something similar - consumer does not care about the "loop" runs now, this is handled by runner.

Reply:
@mpraglowski yup, my plan was to name it `Consumer#process`